<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmRunning</class>
 <widget class="QWidget" name="FrmRunning">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MST - Server running</string>
  </property>
  <widget class="QPushButton" name="btnStop">
   <property name="geometry">
    <rect>
     <x>258</x>
     <y>260</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>btnStop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnToggleConfig">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>btnToggleConfig</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>124</width>
     <height>19</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="lblState">
      <property name="text">
       <string>lblState</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblState2">
      <property name="text">
       <string>lblState2</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>311</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListView" name="listConfigurations"/>
    </item>
    <item>
     <widget class="QTextEdit" name="txtBox">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="txtLine"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnSend">
        <property name="maximumSize">
         <size>
          <width>51</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
